mail/views_test.py

```diff
@@ -636,3 +641,89 @@ def test_send_financial_aid_view_improperly_configured(self):
         with patch('mail.views.MailgunClient.send_batch', side_effect=ImproperlyConfigured):
             resp = self.client.post(self.url, data=self.request_data, format='json')
         assert resp.status_code == status.HTTP_500_INTERNAL_SERVER_ERROR
+
+
+@ddt.ddt
+class EmailBouncedViewTests(APITestCase, MockedESTestCase):
+    """Test email bounce view web hook"""
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.url = reverse('email_bounced_view')
+
+    def test_email_bounced_hook_api_fail(self):
+        """Test that webhook api returns status code 403"""
+        resp_post = self.client.post(
+            self.url,
+            data={
+                "event": "bounced",
+                "recipient": "c@example.com",
+                "error": "Unable to send email"
+            }
+        )
+        # api returns status code 403 because signature is not provided
+        assert resp_post.status_code == status.HTTP_403_FORBIDDEN
+
+    @override_settings(MAILGUN_KEY="key-12345")
+    @ddt.data(
+        ("d717895b90e49108c74df5e0cecf0d80b20e2fb2ed836bfa9209b723d57b2e77", status.HTTP_200_OK),
+        ("f717895b90e49108c74df5e0cecf0d80b20e2fb2ed836bfa9209b723d57b2e88", status.HTTP_403_FORBIDDEN),
+    )
+    @ddt.unpack
+    def test_email_bounced_hook_api_success(self, signature, status_code):
+        """Test that webhook api returns status code 200 when valid data"""
+        resp_post = self.client.post(
+            self.url,
+            data={
+                "event": "bounced",
+                "recipient": "c@example.com",
+                "error": "Unable to send email",
+                "timestamp": 1507117424,
+                "token": "43f17fa66f43f64ee7f6f0927b03c5b60a4c5eb88cfff4b2c1",
+                "signature": signature
+            }
+        )
+        # api returns status code 200 when signature is valid
+        assert resp_post.status_code == status_code
+
+    @override_settings(MAILGUN_KEY="key-12345")
+    @ddt.data(
+        ("d717895b90e49108c74df5e0cecf0d80b20e2fb2ed836bfa9209b723d57b2e77", True),
+        ("f717895b90e49108c74df5e0cecf0d80b20e2fb2ed836bfa9209b723d57b2e88", False),
+    )
+    @ddt.unpack
+    def test_permission(self, signature, has_permission):
```

Comment:
This test looks like it's testing the same code as the previous test. Can you remove it?